### PR TITLE
Blob repository related comment improvements

### PR DESCRIPTION
**Pull request by DiannaHohensee**

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/repositories/RepositoriesService.java [hunk 1]
```diff
@@ -69,7 +69,15 @@
 import static org.elasticsearch.snapshots.SearchableSnapshotsSettings.SEARCHABLE_SNAPSHOTS_REPOSITORY_UUID_SETTING_KEY;
 
 /**
- * Service responsible for maintaining and providing access to snapshot repositories on nodes.
+ * Service responsible for maintaining and providing access to multiple repositories.
+ *
+ * The elected master creates new repositories on request and persists the repository information in the cluster state. The cluster state
```
Comment (DaveCTurner) [inline server/src/main/java/org/elasticsearch/repositories/RepositoriesService.java:74]: suggest linking to the class here:

```suggestion
 * The elected master creates new repositories on request and persists the {@link RepositoryMetadata} in the cluster state. The cluster state
```

File: server/src/main/java/org/elasticsearch/common/blobstore/support/AbstractBlobContainer.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
 import org.elasticsearch.common.blobstore.BlobPath;
 
 /**
- * A base abstract blob container that implements higher level container methods.
+ * A base abstract blob container that adds some methods implementations that are often identical across many children classes.
```
Comment (DaveCTurner) [inline server/src/main/java/org/elasticsearch/common/blobstore/support/AbstractBlobContainer.java:16]:
```suggestion
 * A base abstract blob container that adds some methods implementations that are often identical across many subclasses.
```

**Review by DaveCTurner — APPROVED** [review]
LGTM (2 superficial/optional suggestions)

**Comment by DiannaHohensee** [comment]
Thanks for taking a look! Applied the two suggestions 👍

**Pull request merged**